docs/Common/Structs/index.md

```diff
@@ -538,7 +539,7 @@
 |stationShortName|String|false|minlength="4" <br> maxlength="7"|Identifies the 4-alpha-character station call sign plus an optional (-FM) extension|
 |stationIDNumber|Common.StationIDNumber|false||Used for network Application. Consists of Country Code and FCC Facility ID.|
 |stationLongName|String|false|minlength="0" <br> maxlength="56"|Identifies the station call sign or other identifying information in the long format.|
-|stationLocation|Common.GPSData|false||Provides the 3-dimensional geographic station location.|
+|stationLocation|Common.GPSData|false|HMI should include only `longitudeDegrees`, `latitudeDegrees`, `altitude` params providing SiSData with `stationLocation` param|Provides the 3-dimensional geographic station location.|
```

Comment:
In the hmi_integration_guidelines, the additional column is mostly used specify the additional parameter arguments specified in the HMI_API.xml file(like array, minvalue, maxvalue, etc.). Might be better to put this in the description column
```suggestion
|stationLocation|Common.GPSData|false||Provides the 3-dimensional geographic station location.<br>HMI should include only `longitudeDegrees`, `latitudeDegrees`, `altitude` params providing SiSData|
```

Reply:
@ShobhitAd, thank you for the suggestion. replaced in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/133/commits/88a435130be3c9afcb4c990e7606b47e6fed2fbd